client/blocks/plan-thank-you-card/style.scss

```diff
@@ -21,7 +21,7 @@
 		}
 	}
 
-	.thank-you-card__button,
+	/*.thank-you-card__button,
```

Comment:
Why comment a bunch of styles when we can remove them? Nothing is lost with a merged PR, we can always restore from here if we wish.

Reply:
True, thanks for catching that. Since this deals with the dotcom experience, I carefully commented out styles but didn't actually delete them. Done in https://github.com/Automattic/wp-calypso/pull/28911/commits/f6af194f3c17da6187a14059d91c8b9d45ad7b3f

Reply:
![](https://cldup.com/7WM__pYdDV.png)

😍😍😍